plugins/out_es/es.c

```diff
@@ -453,6 +456,37 @@ static int elasticsearch_format(struct flb_config *config,
         msgpack_pack_str_body(&tmp_pck, time_formatted, s);
 
         es_index = ctx->index;
+        if (ctx->ra_index) {
+            flb_sds_t v = flb_ra_translate(ctx->ra_index,
+                    (char *) tag, tag_len,
+                    map, NULL);
+            if (v) {
+                len = flb_sds_len(v);
```

Comment:
either dynamically allocating the index so it is not truncated or returning a warning or error might be better idea. The limit should also be checked against the actual size of `index_value`, (sizeof()-1), instead of a hardcoded value.